eks-anywhere-common/Testers/StormForge/stormforge-test-cronjob.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: stormforge-test-cronjob
+  namespace: stormforge
+spec:
+  schedule: "30 */6 * * *" # Every six hours, at half-past.
```

Comment:
Sorry i forgot to mention. Lets make it faily pls.

Reply:
Got it, changed to `0 8 * * *` (daily @ 8:00am). That work?

Reply:
Oh, and I added some (small) resource requests to the test job while I was looking at it. The rebase was just to keep the commit history tidy.